apps/roboshield/src/lib/payload/index.ts

```diff
@@ -1,17 +1,25 @@
-import payload from "payload";
-import { ByIDOptions } from "payload/dist/collections/operations/local/update";
-import { Options } from "payload/dist/globals/operations/local/findOne";
-import { PaginatedDocs } from "payload/database";
+import { getPayload } from "payload";
+import configPromise from "../../../payload.config";
+import { PaginatedDocs } from "payload";
 import { Config, Page, SettingsSite } from "@/root/payload-types";
 
 export type CollectionConfig = keyof Config["collections"];
 export type CollectionItemTypes = Config["collections"][CollectionConfig];
 export type GlobalConfig = keyof Config["globals"];
 
+// Define a simple type for options requiring an ID
+interface OptionsWithID {
+  id: string | number;
+  [key: string]: any; // Allow other properties
+}
+
 async function findPage(
   slug: string,
-  options?: Partial<ByIDOptions<CollectionConfig>>,
+  options?: Partial<Record<string, any>>,
 ): Promise<PaginatedDocs<Page>> {
+  const payload = await getPayload({
+    config: configPromise,
+  });
```

Comment:
I see that this function to bootstrap payload is reused across multiple places. Perhaps we can have just one function then re-use it?